src/management/OrganizationsManager.js

```diff
@@ -0,0 +1,818 @@
+ * @param   {Object}    data                  add members data
+ * @param   {Array}     data.members          Array of user IDs
+ * @param   {Function}  [cb]                  Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.removeMembers = function(params, data, cb) {
+  data = data || {};
+  params = params || {};
+
+  if (!params.id) {
+    throw new ArgumentError('The organization ID passed in params cannot be null or undefined');
+  }
+  if (typeof params.id !== 'string') {
+    throw new ArgumentError('The organization ID has to be a string');
+  }
+
+  if (cb && cb instanceof Function) {
+    return this.members.delete(params, data, cb);
+  }
+
+  return this.members.delete(params, data);
+};
+
+/**
+ **** Organization Invites
+ */
+
+/**
+ * Get Invites in a Organization
+ *
+ * @method   getInvitations
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params = {id : 'ORGANIZATION_ID'}
+ * @example <caption>
+ *   This method takes an organization ID and returns the invites in an Organization
+ * </caption>
+ *
+ * management.organizations.getInvitations( {id : 'ORGANIZATION_ID'}, function (err, invites) {
+ *   console.log(invites);
+ * });
+ *
+ * @param   {String}    [params.id]   Organization ID
+ * @param   {Function}  [cb]                Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.getInvitations = function(params, callback) {
+  return this.invitations.getAll(params, callback);
+};
+
+/**
+ * Get an Invitation in a Organization
+ *
+ * @method    getInvitation
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params = {id : 'ORGANIZATION_ID', invitation_id: 'INVITATION_ID'}
+ * @example <caption>
+ *   This methods takes the organization ID and user ID and returns the invitation
+ * </caption>
+ *
+ * management.organizations.getInvitation({id : 'ORGANIZATION_ID', invitation_id: 'INVITATION_ID'}, function (err, invite) {
+ *   console.log(invite);
+ * });
+ *
+ * @param   {String}    [organization_id]   Organization ID
+ * @param   {Function}  [cb]                Callback function.
+ *
+ * @return  {Promise|undefined}
+ */
+OrganizationsManager.prototype.getInvitation = function(params, callback) {
+  return this.invitations.get(params, callback);
+};
+
+/**
+ * Create an invitation in an organization
+ *
+ * @method    createInvitation
+ * @memberOf  module:management.OrganizationsManager.prototype
+ *
+ * @example
+ * var params =  { id :'ORGANIZATION_ID'};
+ * var data = {
+ *   client_id: CLIENT_ID,
+ *   invitee: { email: 'invitee@example.com' },
+ *   inviter: { name: 'John Doe' }
+ * };
+ *
+ * management.organizations.createInvitation(params, data, function (err) {
+ *   if (err) {
+ *     // Handle error.
+ *   }
+ * });
+ *
+ * @param   {String}    params.id             ID of the Organization.
+ * @param   {Array}     data                  Invitation data
```

Comment:
Optional: Nice to have to add a mention of all possible data.XXX attributes allowed (just like you did for tickets endpoints). See here - https://auth0team.atlassian.net/wiki/spaces/USER/pages/1000735499/User+Invitation#api2 for allowed attributes.